{"usn":"USN-1965-1","special":"The problem can be corrected by updating your system to the following package versions:\nIn general, a standard system update will make all the necessary changes.","update":{"13.04":[{"version":"0.13-2ubuntu3.1","package":"python-openssl"},{"version":"0.13-2ubuntu3.1","package":"python3-openssl"}],"10.04":[{"package":"python-openssl","version":"0.10-1ubuntu0.1"}],"12.10":[{"version":"0.13-2ubuntu1.1","package":"python-openssl"},{"version":"0.13-2ubuntu1.1","package":"python3-openssl"}],"12.04":[{"version":"0.12-1ubuntu2.1","package":"python-openssl"}]},"CVEs":["CVE-2013-4314"],"threat":["An attacker could exploit this to perform a man in the middle attack to view sensitive information or alter encrypted communications.."],"info":"pyopenssl vulnerability","date":"September 23, 2013"}